app/views/applicant_mailer/assignment.html.haml

```diff
@@ -0,0 +1,12 @@
+%p
+  Dear #{full_name(@applicant)},
+
+%p
+  You have been assigned as an applicant on the submission
+  = surround '"', ',"' do
```

Comment:
I don't think you need that opening comma.

Reply:
You do. This is an appositive.